mkt/site/tests/__init__.py

```diff
@@ -754,17 +754,15 @@ def req_factory_factory(url='', user=None, post=False, data=None, **kwargs):
 
 def user_factory(**kw):
     """
-    If not provided, email will be 'factoryuser<number>@mozilla.com' and
-    username will be a <random 32 hex string>.
+    If not provided, email will be 'factoryuser<number>@mozilla.com'.
     If email has no '@' it will be corrected to 'email@mozilla.com'
     """
     global user_factory_counter
-    username = kw.pop('username', '%030x' % random.randrange(16 ** 30))
     email = kw.pop('email', 'factoryuser%d' % user_factory_counter)
     if '@' not in email:
         email = '%s@mozilla.com' % email
 
-    user = UserProfile.objects.create(username=username, email=email, **kw)
```

Comment:
should this function be generating a fxa id instead now?  (the 32 hex string was a generated version of it)